packaging/install/cluster-operator/023-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -74,3 +74,10 @@ rules:
       - get
       - patch
       - update
+  - apiGroups:
+      - "kafka.strimzi.io"
+    resources:
+      # The Cluster Operator needs deletion for KafkaRebalance only (during auto-rebalancing)
+      - kafkarebalances
+    verbs:
+      - delete
```

Comment:
MAybe you can shift the order to have this next to the other kafka-rebalance rules? HAving it here at the end makes it confusing as hell thinking _why do you need only delete_ and not realizing that the other rights are elsewhere.

Reply:
Sure.